docs/developers/serializers.rst

```diff
@@ -0,0 +1,187 @@
+.. _serializers:
+
+Serializers
+===========
+
+You can customize the serialization to and from Stores in Pootle.
+
+This allows you to make formatting or other changes before saving to disk, and
+conversely you can make changes to files before importing into Pootle.
+
+Serializers and deserializers are configured for a project. See the
+`django-admin`:"project_serializers" command for more information on configuring projects.
+
+To add your own serializers of deserializers you must add your own django application,
+and include it in your `INSTALLED_APPS` setting.
+
+This document explains what files you will need in your application to add serializers
+or deserializers.
+
+
+.. _serializers#custom-serializer:
+
+Adding a custom serializer
+--------------------------
+
+Lets add a `CustomSerializer` in a file called `serializers.py`.
```

Comment:
@ta2-1 i havent refactored the docs since i landed the plugin doc page.

Once i do, ill tidy this up and link to the relevant sections on the plugin page.